app/views/home/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if signed_in? %>
```

Comment:
please remove this condition because this view will always be rendered for the signed user

Reply:
Done